data/Raj210Kumari.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Rajnandani Kumari",
+  "type": "personal",
+  "bio": "**Frontend Developer** Skilled in MERN stack who focuses on writing clean, elegant and efficient code.",
+  "links": [
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/Raj210Kumari",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Checkout my portfolio",
+      "url": "http://raj210kumari.github.io/",
+      "icon": "FaWeb"
```

You could use `FaGlobe` for this, as `FaWeb` is not a valid icon as @senali-d said.
```suggestion
      "icon": "FaGlobe"
```